### ISSUE TITLE
octo.exe list-latestdeployments with --outputformat=JSON flag doesn't provide valid JSON

### ISSUE THREAD

**Issue by alexrolleyoctopus** [OctopusDeploy/Issues#5224]
# Prerequisites

- [x] I have verified the problem exists in the latest version
- [x] I have searched [open](https://github.com/OctopusDeploy/Issues/issues) and [closed](https://github.com/OctopusDeploy/Issues/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed) issues to make sure it isn't already reported
- [x] I have written a descriptive issue title
- [x] I have linked the original source of this report
- [x] I have tagged the issue appropriately (area/*, kind/bug, tag/regression?)

# The bug

When requesting a JSON output of latest-deployments using Octo.exe the output is not valid JSON. Side effect of [this PR](https://github.com/OctopusDeploy/OctopusClients/pull/353)

## What I expected to happen

Using the --outputformat=JSON should provide valid JSON.

## Steps to reproduce

1. Use any version of Octopus Tools/Octo.exe later than 4.43.0
2. Pass the following parameters
```
octo list-latestdeployments --server=http://<octopus-server> --apikey=API-KEYxxxxxxxxxx --outputformat=json
```
3. Note that the first line of the returned output is not valid JSON
```
Detected automation environment: "NoneOrUnknown"
```

### Screen capture

<img width="1250" alt="screen shot 2019-01-17 at 4 15 30 pm" src="https://user-images.githubusercontent.com/29847178/51299174-25d83b80-1a73-11e9-9885-5803ff4a37c8.png">

## Affected versions

**Octopus Tools: 4.43.0 > Latest** 

## Workarounds

Use a pre 4.43.0 version of Octo.exe/Octopus Tools

## Links

[Initial Report](https://secure.helpscout.net/conversation/756204057/38029?folderId=1465198)

**Comment by JoshStorkey** [comment]
We are also having this issue when trying to pipe the output using
`list-releases --project $Project --server $Server --apiKey $APIKey --outputFormat=json | ConvertFrom-Json; | ConvertFrom-Json;`

>  + Invalid JSON primitive: Detected.

**Comment by salamoun** [comment]
Same for me, I'm getting Detected automation environment: "NoneOrUnknown"

**Comment by slewis74** [comment]
This turned out to be due to something in the way we handle logging between Octo.exe and Octopus.Client. Some logging got added in some new code added to allow feature detection in the Spaces version of server but Octopus.Client wasn't handling the "muting" correctly when the json formatting was in use.

We've just published `5.2.1` with a fix. Sorry for the inconvenience caused.
